lib/build-parent-toc.js

```diff
@@ -77,13 +77,13 @@ function getChildren(folder) {
 }
 
 function processFile(path, file, mode) {
-  let toc = transform(fs.readFileSync(path + '/' + file, 'utf8'), mode, 12).toc;
+  let toc = transform(fs.readFileSync(path + '/' + file, 'utf8'), mode, 12, "", true).toc;
```

Comment:
More of a general question, but are passing in the empty string as the 2nd to last argument for a specific reason?  Only reason I'm asking is it looks like the `processAll` parameter behaves as a boolean in the `transform` implementation.

Reply:
AFAIK, the args are positional and we wanted to pass the `notitle` arg (the last arg) as `true` so that we wouldn't have to strip the title back out on line 86 below.
So then we had to pass _something_ for the `title` arg (which inexplicably comes before the `notitle` flag... 🤦  )

Reply:
That makes sense, I actually counted wrong (completely off 🤦‍♂️ ) on my initial comment for the positional parameters, and I thought we were passing in the empty string for a different parameter.